src/app/modal/components/lose-changes/lose-changes.component.html

```diff
@@ -0,0 +1,14 @@
+<div mat-dialog-content lose-changes>
+    <h1 class="govuk-heading-m title">Unsaved changes</h1>
+    <span class="govuk-hint">
+      If you leave this page, your changes will be lost.
+    </span>
+  </div>
+
+  <button class="govuk-button" data-module="govuk-button" (click)="onLoseChangesClick()" cdkFocusInitial>
```

Comment:
Do we need any id on that if we were to test it?

Reply:
ids added to both buttons